index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Isaac Jung's Resume</title>
+    <link rel='stylesheet' href='styles.css' />
+</head>
+
+<body>
+    <div class="page u-clearfix">
+        <main>
+            <div class="content01">
+                <h1>Graphic Designer & Sound Artist</h1>
+                <p><u>Isaac Jung</u> is based in Seoul, Republic of Korea.
```

Comment:
2 things to point out here:

1. While the `u` element _does_ create an underline effect in browsers, that's not it's **meaning**. See the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u#Usage_notes) for the meaning and valid uses cases of the `u` element and consider if it is appropriate here.
2. Underlining for presentation purposes can be tricky in web design because user expectations are that underlines mean links. So, be thoughtful about how you include underlines in your web designs.